aubrey_transcription/utils.py

```diff
@@ -20,7 +20,39 @@ def find_files(pairpath):
     normalized_path = os.path.normpath(full_path)
     if not os.path.isdir(normalized_path):
         return []
-    return os.listdir(normalized_path)
+    files = os.listdir(normalized_path)
+    try:
+        # Sort the files numerically by the fileset number
+        return sorted(files, key=lambda f: int(f.split('-')[0].split('_')[-1]))
+    except ValueError:
```

Comment:
Since a setting defines FILENAME_PATTERN, perhaps we should use that to get the fileset number.
```suggestion
        return sorted(files, key=lambda f: int(decrypt_filename(f)['fileset']))
    except (ValueError, KeyError):
```

Reply:
Ah, yeah, that seems like a much better way of doing it, I'll use that.